ruby/README.md

```diff
@@ -0,0 +1,43 @@
+# Intro
```

Comment:
Add *Ruby* in [list](https://github.com/TinkoffCreditSystems/voicekit-examples#language-specific-instructions) with languages examples, please.

Reply:
Sure, done